openpype/hosts/maya/api/pipeline.py

```diff
@@ -166,7 +169,7 @@ def _before_scene_save(return_code, client_data):
     # in order to block the operation.
     OpenMaya.MScriptUtil.setBool(return_code, True)
 
-    avalon.api.emit("before_save", [return_code, client_data])
+    emit_event("before.save")
```

Comment:
The previous logic allowed the `before.save` event to use the return code arguments to block Maya from allowing to save. This new logic doesn't. Are we aware this change removes that possibility and introduces a new limitation?

Reply:
It can be changed to:
```
emit_event(
    "before.save",
    {
        "return_code": return_code,
        "client_data": client_data
    }
)
```
So callback can do:
```
def my_callback(event):
    return_code = event.get("return_code")
    client_data= event.get("client_data")
    ...
```
It was not used in the code so I've removed it, do you use it?

Reply:
I personally didn't no - just wondering whether the limitation would be solvable if we needed it.